src/com/sap/piper/JenkinsUtils.groovy

```diff
@@ -19,6 +19,31 @@ static boolean hasTestFailures(build){
     return action && action.getFailCount() != 0
 }
 
+boolean addWarningsNGParser(String id, String name, String regex, String script, String example = ''){
+    def classLoader = this.getClass().getClassLoader()
```

Comment:
What is the reason for loading the class via class loader magic? Why is it not possible/desired to load create an instance via the normal way (use constructor)? Is the class only available at runtime, but not on compile time? If "yes": would it be an option to adjust the depencencies in order to make transparent what we use?

Reply:
I chose the class loader to avoid dependencies when they are not used. When I import the class it (and with it the plugin) is needed as soon as the `JenkinsUtils` class is used.